d3b_utils/aws_bucket_contents.py

```diff
@@ -0,0 +1,71 @@
+import boto3
+import pandas as pd
+import re
+
+
+def fetch_aws_bucket_obj_info(bucket_name, search_prefixes=None):
+    """
+    List the objects of a bucket and the size of each object. Returns a
+    dict.
+
+    :param bucket_name: The name of the bucket.
+    :param path_to_files: path to the specific folder in the given
+        bucket where the files you want to list are. Automatically drops
+        leading '/'. Include a '/' at the end of folder names. Should look
+        like 'path/to/files/'. This can also be any iteratable.
+    """
+    search_prefixes = search_prefixes or ""
+
+    if isinstance(search_prefixes, str):
+        search_prefixes = [search_prefixes]
+
+    # Drop leading non-word characters from prefixes
```

Comment:
this only drops one leading forward slash, not all non-word characters

Reply:
I've been thinking about this - I think I only want it drop slashes. If it dropped all non-word characters, this function wouldn't be able to get the contents of a folder that starts with `.`. changed the comment to clear that up